configure

```diff
@@ -4049,6 +4074,28 @@ fi
 define_in_config_if_yes "$_opengl" "USE_OPENGL"
 define_in_config_if_yes "$_opengles" "USE_GLES"
 
+# Check if Raspberry Pi dispmanx has been activated. 
+if test "$_dispmanx" = "yes" ; then
+		echocheck "DispmanX graphics"
+		_use_dispmanx=no
+		DISPMANX_CXXFLAGS="-I$RPI_ROOT/opt/vc/include -I$RPI_ROOT/opt/vc/include/interface/vmcs_host/linux/ -I$RPI_ROOT/opt/vc/include/interface/vcos/pthreads -I$RPI_ROOT/opt/rpi_root/usr/include/SDL"
+		DISPMANX_LIBS="--sysroot=$RPI_ROOT -L$RPI_ROOT/usr/lib -L$RPI_ROOT/opt/vc/lib -lbcm_host -lvcos -lvchiq_arm"
+		cat > $TMPC << EOF
+#include <bcm_host.h>
+
+		int main(int argc, char *argv[]) {
+	bcm_host_init();
+}
+EOF
+		cc_check $DISPMANX_CXXFLAGS $DISPMANX_LIBS && _use_dispmanx=yes
+		if test "$_use_dispmanx" = "yes"; then
+			CXXFLAGS="$CXXFLAGS $DISPMANX_CXXFLAGS"
+			LIBS="$LIBS $DISPMANX_LIBS"
+			DEFINES="$DEFINES -DRASPBERRYPI"
+			add_line_to_config_mk 'RASPBERRYPI = 1'
```

Comment:
Is this enabling the whole RaspberryPI backend? If so that looks really strange and should rather be decided based on the _backend setting, i.e. --backend.